jupyterlab_automl/src/components/list_resources_panel.tsx

```diff
@@ -417,32 +526,37 @@ export class ListResourcesPanel extends React.Component<Props, State> {
   }
 
   private async refresh() {
-    try {
-      this.setState({ isLoading: true });
-      await Promise.all([this.getDatasets(), this.getModels()]);
-      this.setState({ hasLoaded: true });
-    } catch (err) {
-      console.warn('Error retrieving datasets', err);
-    } finally {
-      this.setState({ isLoading: false });
-    }
+    this.setState({ isLoading: true });
+    await Promise.all([
+      this.getDatasets(),
+      this.getModels(),
+      this.getPipelines(),
+    ]);
+    this.setState({ hasLoaded: true });
+    this.setState({ isLoading: false });
   }
```

Comment:
should the refresh just update the resource type that the user is on (i.e. datasets, models, or training pipelines) to save time?